libs/mimir2/src/adapters/secondary/elasticsearch/mod.rs

```diff
@@ -8,7 +8,7 @@ pub mod remote;
 pub mod storage;
 
 #[derive(Clone)]
-pub struct ElasticsearchStorage(Elasticsearch);
+pub struct ElasticsearchStorage(pub Elasticsearch);
```

Comment:
Couldn't we manage to only allow access from `mimir2`?

```suggestion
pub struct ElasticsearchStorage(pub(crate) Elasticsearch);
```